<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMainWindow</class>
 <widget class="QMainWindow" name="WMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QProjectTreeView" name="treeView"/>
      <widget class="QMdiArea" name="mdiArea">
       <property name="viewMode">
        <enum>QMdiArea::TabbedView</enum>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="tabsMovable">
        <bool>true</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_project_as"/>
    <addaction name="actionClose_project"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_data_source"/>
    <addaction name="actionNew_plot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionSave_project_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save project as</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionNew_data_source">
   <property name="text">
    <string>New data source</string>
   </property>
  </action>
  <action name="actionNew_plot">
   <property name="text">
    <string>New plot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QProjectTreeView</class>
   <extends>QTreeView</extends>
   <header>qprojecttreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
